vermouth/tests/test_molecule.py

```diff
@@ -889,17 +889,36 @@ def attribute_dict(draw, min_size=0, max_size=None):
     return draw(st.dictionaries(keys, values, min_size=min_size, max_size=max_size))
 
 
+@st.composite
+def parameter_effectors(draw):
+    possible_effectors = [
```

Comment:
Pass them as a (default) argument?

Reply:
In principle, why not. But I do not see an immediate use for it. I'll reconsider if something comes up.